packages/mock-block-dock/src/hook-portals/text/toolbar.tsx

```diff
@@ -1,24 +1,77 @@
+import { ReactElement, useEffect, useRef } from "react";
+import { createPortal } from "react-dom";
+import { Editor, Range } from "slate";
+import { useFocused, useSlate } from "slate-react";
+
 import { MarkButton } from "./mark-button";
 
+const Portal = ({ children }: { children: ReactElement }) => {
+  return createPortal(children, document.body);
+};
```

Comment:
Could you just call `createPortal` in `Toolbar`?

Reply:
Yes - https://github.com/blockprotocol/blockprotocol/pull/707/commits/8cbb862915ec99adce6ca230d209ec5a07b5a8af